content/en/guides/internals-glossary/context.md

```diff
@@ -147,6 +147,35 @@ Response from the Node.js server. If Nuxt is used as a middleware, the res objec
 
 Use this method to update `__NUXT__` variable rendered on client-side, the `fn` (can be asynchronous) is called with `{ Components, nuxtState }`, see [example](https://github.com/nuxt/nuxt.js/blob/cf6b0df45f678c5ac35535d49710c606ab34787d/test/fixtures/basic/pages/special-state.vue).
 
+### beforeSerialize
+
+`beforeSerialize(fn)` (_Function_)
+
+Use this method to update `__NUXT__` variable rendered on client-side, the `fn` (must be synchronous) is called with `nuxtState` as argument. This method is called inside the `rendered` method of Vue SSR, allowing you to use it in components.
+
+Not this usage is advanced and mostly used for Nuxt modules.
```

Comment:
Maybe add a notice is available from 2.16+?

Reply:
Updated, thanks for pointing that out!